easybuild/easyconfigs/g/GffCompare/GffCompare-0.10.1-foss-2016b.eb

```diff
@@ -0,0 +1,42 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'MakeCp'
+
+name = 'GffCompare'
+version = '0.10.1'
+git_commit_id = '305b7f80eb33171d39692bc69ea487c1ef9e2275'
+
+homepage = 'https://github.com/gpertea/%(namelower)s'
+description = """GffCompare provides classification and reference annotation mapping and
+ matching statistics for RNA-Seq assemblies (transfrags) or other generic GFF/GTF files."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'pic': True}
+
+source_urls = [
+    'https://github.com/gpertea/%(namelower)s/archive/',
+    'https://github.com/gpertea/gclib/archive/',
+]
+sources = [
+    'v%(version)s.tar.gz',
+    '%s.tar.gz' % git_commit_id,
```

Comment:
rename the `gclib` tarball on download, as follows:

```python
sources = [
    'v%(version)s.tar.gz',
    {'filename': 'gclib-20171013.tar.gz', 'download_filename': '305b7f8.tar.gz'},
]

Reply:
@boegel I changed this.